app.js

```diff
@@ -0,0 +1,262 @@
+'http://www.telegraph.co.uk/food-and-drink/cocktails/10-classic-mad-men-era-cocktails/mint-julep/', 'http://www.telegraph.co.uk/food-and-drink/cocktails/10-classic-mad-men-era-cocktails/brandy-alexander/',
+'http://www.drinksmixer.com/drinkn1m2540.html',
+'http://www.foodnetwork.com/recipes/rachael-ray/white-sangria-recipe', 'http://www.seriouseats.com/recipes/2012/01/cuba-libre-rum-coke-cocktail-recipe.html',  'http://www.liquidgolddrinks.com/cocktails/prosecco-cocktails/raspberry-prosecco-bellini/',
+'https://www.thespruce.com/rum-and-coke-recipe-760560',
+'http://allrecipes.com/recipe/234754/almond-joy-cocktail/',
+'http://www.seriouseats.com/recipes/2014/08/suze-cava-st-germain-cocktail-aperitif-drink.html',
+'http://www.seriouseats.com/recipes/2012/10/gramercy-tavern-man-about-town-cocktail-rye-boulevardier-cynar-recipe.html',
+'http://www.drinksmixer.com/drink7265.html',
+'http://www.barnonedrinks.com/drinks/t/the-spitfire-7751.html',
+'http://fireballwhisky.com/recipes/angry-balls/',
+'http://imbibemagazine.com/chrysanthemum-cocktail-recipe/',
+'http://www.seriouseats.com/recipes/2011/02/the-obituary-cocktail-gin-absinthe-vermouth.html',
+'http://www.seriouseats.com/recipes/2008/06/traditional-old-fashioned-recipe.html',
+'http://www.seriouseats.com/recipes/2013/02/frisco-sour-simple-rye-cocktail-lemon-benedictine-recipe.html',
+'http://www.seriouseats.com/recipes/2008/09/boulevardier-recipe.html',
+'http://www.seriouseats.com/recipes/2015/03/gold-rush-bourbon-honey-cocktail-drink-recipe.html',
+'http://www.seriouseats.com/recipes/2012/02/drinking-in-season-grapefruit-ginger-sparkling-wine-cocktail-recipe.html',
+'http://www.seriouseats.com/recipes/2014/03/simple-cocktail-blushing-betty-bourbon-grapefruit-easy-drink-recipe.html',
+'http://www.seriouseats.com/recipes/2008/05/daiquiri-recipe.html',
+'https://www.popsugar.com/food/Hot-Cold-Butterbeer-Recipe-380247',
+'https://www.popsugar.com/food/Pomegranate-Champagne-Cocktail-26435241',
+'http://youngaustinian.com/2014/02/13/modern-austen-blood-orange-rickey/',
+'https://www.popsugar.com/food/Salty-Dog-Cocktail-Recipe-2674478',
+'https://www.popsugar.com/food/Coffee-Martini-37944195']
+
+
+var cocktailRecipeBox = [];
+console.log(cocktailRecipeBox);
+
+
+// Constructor Function to create Cocktail objects
+function Cocktail(name, imagePath, ingredient1, ingredient2, ingredient3, link){
+  this.name = name;
+  this.imagePath = imagePath;
+  this.ingredient1 = ingredient1;
+  this.ingredient2 = ingredient2;
+  this.ingredient3 = ingredient3;
+  this.link = link;
+  cocktailRecipeBox.push(this);
+}
+// This IIFE Creates the cocktail objects on load
+(function() {
+
+  for (var i in cocktailNames){
+    var newCocktail = new Cocktail(cocktailNames[i], 'assets/assetspng/' + cocktailNames[i] + '.png', ingredient1Array[i], ingredient2Array[i],
```

Comment:
No need for a variable here. You can just use the `new Cocktail()` syntax without assigning it to a var, and accomplish the same thing.